lib/commands/file-movement.js

```diff
@@ -210,7 +222,7 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
     await mkdirpDevice(service, path.dirname(relativePath));
     const stream = await service.createWriteStream(relativePath, { autoClose: true });
```

Comment:
are different container types supported here?

Reply:
Yeah. The container types are used in `createAfcClient` and `createService`. The rest is uniform like no container type